WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -69,6 +70,7 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
     private var updateEnabledSubscription: AnyCancellable?
     private var newVariationsPriceSubscription: AnyCancellable?
     private var productImageStatusesSubscription: AnyCancellable?
+    private var updateBlazeEligibility: AnyCancellable?
```

Comment:
Nit: we should add the `subscription` suffix to clarify that this property holds the reference of a Combine subscription.

```suggestion
    private var blazeEligibilitySubscription: AnyCancellable?
```